chardonnay/genesis.json

```diff
@@ -22,12 +22,15 @@
   },
   "app_hash": "",
   "app_state": {
-    "allocation": {
+    "auction": {
```

Comment:
Are you sure you generated your init genesis from v6? `auction` doesn't exist yet.